Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/StartupHandler.java

```diff
@@ -36,10 +40,32 @@ public static void HandleInit(FragmentActivity parent)
     String[] gamesToLaunch = getGamesFromIntent(parent.getIntent());
     if (gamesToLaunch != null && gamesToLaunch.length > 0)
     {
+      // Open permission dialog, the game will be opened on the next attempt
+      if ((Build.VERSION.SDK_INT <= 29) && !PermissionsHandler.hasWriteAccess(parent))
```

Comment:
Is there a reason why it would not be acceptable to run Dolphin on old Android without write external storage permission? The rest of Dolphin handles it just fine, by using Android/data/ as the User folder.

Reply:
When the path is legacy then it didn't work for me without the permission.

Note that VR Android devices have quite often very modified Android API. It doesn't behave as it should.

Reply:
> When the path is legacy then it didn't work for me without the permission.

That's true. But if that's the problem, it would be better to check whether the path is a legacy path than to check which version of Android you're on.